examples/django/oauth_app/slack_datastores.py

```diff
@@ -85,13 +125,15 @@ def find_installation(
             t_id = None
         if user_id is None:
             rows = (
-                SlackInstallation.objects.filter(enterprise_id=e_id)
+                SlackInstallation.objects.filter(client_id=self.client_id)
```

Comment:
fixed a minor bug where multiple Slack apps in a single database table does not properly work